<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-GNQ7IEL" failures="1" tests="2" name="Default test(failed)" time="0.088" errors="0" timestamp="2022-12-31T21:37:28 IST">
  <testcase classname="testngstudyxml.failedTest" name="d" time="0.019">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testngstudyxml.failedTest.d(failedTest.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- d -->
  <testcase classname="testngstudyxml.failedTest" name="a" time="0.003"/>
  <testcase classname="testngstudyxml.failedTest" name="b" time="0.0">
    <ignored/>
  </testcase> <!-- b -->
  <testcase classname="testngstudyxml.failedTest" name="c" time="0.0">
    <ignored/>
  </testcase> <!-- c -->
</testsuite> <!-- Default test(failed) -->
